packages/cli/src/commands/deploy.ts

```diff
@@ -39,13 +39,18 @@ export default class Deploy extends AuthCommand {
       description: 'force mode',
       default: false,
     }),
+    config: Flags.string({
+      char: 'c',
+      description: 'The Checkly CLI config filename.',
+      allowNo: true,
+    }),
   }
 
   async run (): Promise<void> {
     const { flags } = await this.parse(Deploy)
-    const { force, preview, output } = flags
+    const { force, preview, output, config: configFilename } = flags
     const cwd = process.cwd()
```

Comment:
@clample @nahuelon we should probably change `cwd` to the config file directory, given built relative paths according to check and config files. What do you think?

Reply:
Hmm, that makes sense ➕

Reply:
Good idea

Reply:
@umutuzgur , during the coding I tried lot of options related with that. I also tried to allow to enter a path (without filename) in the `--config` argument (and search for the `checkly.config.[ts,js]`). But, I realized about the paths in the config file and I reverted it. I'll try to change the `cwd` to the config file directory, and see if it's safe enoght.

Reply:
@umutuzgur , this change would require more time to code/test it. Maybe we can move on with the `--config` as it is now and then create a second Issue/PR to modify where the `cwd` should point.

Reply:
@nahuelon it should be together so we don't create temporary breaking change